app/main/forms/award_or_cancel.py

```diff
@@ -0,0 +1,23 @@
+from flask.ext.wtf import Form
+from wtforms import RadioField, validators
+
+
+class AwardOrCancelBriefForm(Form):
+    """Form for the buyer to tell us why they cancel the requirement
```

Comment:
I think this docstring needs tweaking

Reply:
Doh! I've got one unpushed commit locally. Guess what it does :grin: